src/code/bank20.asm

```diff
@@ -2212,79 +2212,50 @@ jr_020_5664:
-    db   $04, $04, $04, $04, $04, $04, $04, $04   ; $57D3
-    db   $00, $00, $00, $00, $00, $00, $01, $01   ; $57DB
-    db   $04, $06, $04, $04, $04, $07, $07, $04   ; $57E3
-    db   $00, $00, $00, $00, $00, $01, $01, $01   ; $57EB
-    db   $04, $06, $04, $04, $04, $07, $07, $04   ; $57F3
-    db   $00, $00, $00, $00, $00, $00, $01, $01   ; $57FB
-    db   $04, $04, $04, $04, $04, $04, $04, $04   ; $5803
-    db   $00, $01, $02, $02, $03, $03, $03, $03   ; $580B
-    db   $00, $00, $00, $04, $02, $07, $00, $00   ; $5813
-    db   $00, $00, $02, $02, $00, $00, $00, $00   ; $581B
-    db   $00, $00, $00, $04, $07, $07, $00, $00   ; $5823
-    db   $07, $07, $01, $01, $00, $00, $04, $04   ; $582B
-    db   $00, $00, $00, $04, $05, $03, $03, $03   ; $5833
-    db   $07, $07, $01, $01, $00, $04, $04, $04   ; $583B
-    db   $00, $00, $04, $04, $03, $03, $03, $03   ; $5843
-    db   $03, $03, $03, $03, $00, $00, $00, $00   ; $584B
-    db   $04, $04, $04, $00, $07, $01, $03, $03   ; $5853
-    db   $03, $03, $03, $03, $00, $00, $02, $00   ; $585B
-    db   $04, $04, $04, $00, $00, $02, $03, $03   ; $5863
-    db   $03, $03, $03, $07, $03, $03, $00, $00   ; $586B
-    db   $04, $04, $04, $00, $04, $04, $03, $03   ; $5873
-    db   $04, $04, $04, $04, $04, $04, $07, $00   ; $587B
-    db   $04, $04, $00, $00, $00, $00, $03, $03   ; $5883
+; The tiles to draw for the visited locations (see OW_ROOM_STATUS_VISITED) on the overworld
+; map, within the outer border defined in WorldMapTilemap
+InnerWorldMapTilemap::
+    db   $27, $6A, $6C, $21, $22, $23, $24, $25, $26, $6A, $FF, $6C, $6A, $6C, $6A, $6C
+    db   $65, $65, $66, $31, $32, $33, $34, $35, $36, $67, $68, $64, $67, $69, $65, $66
+    db   $40, $40, $41, $42, $43, $44, $FA, $FA, $63, $40, $40, $40, $40, $40, $40, $60
+    db   $FF, $FA, $48, $49, $4A, $FA, $FA, $FF, $62, $6D, $6D, $6D, $6D, $6D, $6D, $FF
+    db   $00, $01, $00, $01, $FA, $FF, $5E, $5F, $04, $05, $06, $07, $28, $29, $29, $2A
+    db   $10, $11, $10, $11, $FA, $FA, $6E, $6F, $14, $15, $16, $17, $38, $20, $20, $3A
+    db   $00, $01, $00, $01, $FB, $FF, $FE, $FE, $08, $09, $0A, $0B, $38, $20, $20, $3A
+    db   $10, $11, $10, $11, $FB, $FB, $FE, $FE, $18, $19, $1A, $1B, $48, $49, $49, $4A
+    db   $FB, $FF, $0C, $0D, $40, $40, $40, $40, $FA, $FA, $FF, $58, $0E, $0F, $FA, $FA
+    db   $FB, $FB, $1C, $1D, $FA, $FA, $FA, $FA, $FA, $FA, $FA, $5D, $1E, $1F, $FA, $FA
+    db   $0C, $0D, $0C, $0D, $FB, $FB, $28, $2A, $FA, $FA, $FA, $58, $2D, $2E, $2E, $2F
+    db   $1C, $1D, $1C, $1D, $FB, $56, $61, $4A, $FA, $FA, $59, $5A, $3D, $3E, $3E, $3F
+    db   $FD, $FD, $FD, $FD, $FB, $FB, $FB, $FB, $28, $29, $5B, $FA, $FF, $FF, $54, $54
+    db   $FD, $FD, $FD, $FD, $FB, $FB, $FF, $FB, $38, $30, $3A, $FA, $FF, $FF, $54, $54
+    db   $FD, $FD, $FD, $FF, $FD, $FD, $FB, $FB, $48, $FE, $4A, $FA, $56, $57, $54, $54
+    db   $03, $12, $13, $12, $13, $02, $FF, $FB, $5C, $2B, $FA, $FA, $FA, $FA, $54, $54
```

Comment:
It almost makes me want to define a custom charmap with emojis, so that we could do:

```asm
InnerWorldMapTilemap::
    db  🏰, ⛰, ⛰, 🥚 ; …
```